ext/fs/Cargo.toml

```diff
@@ -25,10 +25,12 @@ deno_io.workspace = true
 deno_path_util.workspace = true
 deno_permissions.workspace = true
 filetime.workspace = true
+getrandom = "0.2"
```

Comment:
Is this used on purpose instead of `rand` crate?

Reply:
Yes, this is what the rand crate uses and the SystemRandom trait needs this lower level function